auth-web/src/views/auth/staff/StaffDashboardView.vue

```diff
@@ -7,53 +7,60 @@
       </p>
     </div>
 
-    <v-card flat class="mb-4 pa-8">
-      <div class="view-header flex-column mb-10">
-        <h2 class="view-header__title">Search Entities</h2>
-        <p class="mt-3 mb-0">
-          Enter the Entity's Incorporation Number below to access their
-          dashboard.
-        </p>
-      </div>
-      <v-expand-transition>
-        <div v-show="errorMessage">
-          <v-alert type="error" icon="mdi-alert-circle" class="mb-0"
-            >{{ errorMessage }} <strong>{{ searchedBusinessNumber }}</strong>
-          </v-alert>
-        </div>
-      </v-expand-transition>
-      <v-form ref="searchBusinessForm" v-on:submit.prevent="search">
-        <v-text-field
-          filled
-          label="Incorporation Number"
-          hint="example: CP0001234"
-          persistent-hint
-          dense
-          req
-          @blur="incorpNumFormat"
-          :rules="incorpNumRules"
-          v-model="businessNumber"
-          id="txtBusinessNumber"
-        >
-        </v-text-field>
-        <v-btn
-          color="primary"
-          class="search-btn mt-0"
-          type="submit"
-          depressed
-          :disabled="!isFormValid()"
-          :loading="searchActive"
-          >Search</v-btn
-        >
-      </v-form>
-
-      <template>
-        <IncorporationSearchResultView
-          :isVisible="canViewIncorporationSearchResult"
-          :affiliatedOrg="affiliatedOrg"
-        ></IncorporationSearchResultView>
-      </template>
-    </v-card>
+    <v-row class="ma-0 mb-4" no-gutters>
+      <v-col class="pr-2" cols="6">
+        <v-card class="pa-8" flat style="height: 100%;">
```

Comment:
can we add this a component and import here.

Reply:
This is the existing code so I'd rather not change this / spend any extra time on it in this sprint. All I've done is put it inside the row/col